charts/sonarqube-dce/README.md

```diff
@@ -215,64 +215,83 @@ kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/late
 
 ```
 
+## Secure the communication within the cluster
+
+In order to secure the communication between Application and Search nodes, you need to set both `nodeEncryption.enabled` and `searchNodes.searchAuthentication.enabled` to `true`.
+
+In a secured cluster, Elasticsearch nodes use certificates to identify themselves when communicating with other nodes. You need to generate a Certificate Authority (CA) together with a certificate and private key for the nodes in your cluster. Furthemore, you need to specify the Search nodes' hostnames that will be added as DNS names in the Subject Alternative Name (SAN).
+
+As an example, let's assume that your cluster has three search nodes with the release's name set to "sq" and the chart's name set to "sonarqube-dce". You will need to add the following DNS names in the SAN.
```

Comment:
we might want to add that there is the namespace involved also ? and clearly show what field does which ?

because I think the pattern is release-sonarqube-dce-search-number.release-sonarqube-dce-search.namespace.svc.cluster.local.

which maps to pod.service.namespace.svc.cluster.local ?

Also we should specify that the last one is mandatory and is just the service, which is releaseName-sonarqube-dce-search

Reply:
and after doing it i think we should even provide a detailed tutorial on how to generate the cert